shared/actions/people.tsx

```diff
@@ -95,7 +96,7 @@ const dismissAnnouncement = (_, action: PeopleGen.DismissAnnouncementPayload) =>
 
 const markViewed = () =>
   RPCTypes.homeHomeMarkViewedRpcPromise().catch(err => {
-    if (networkErrors.includes(err.code)) {
+    if (networkErrorCodes.includes(err.code)) {
```

Comment:
lets actually make a isNetworkError in util/error that does this so you can do
`if (isNetworkError(err))`

Reply:
can't go in `util/error` because of a circular import `rpc-gen > engine/require > engine/index > util/errors > rpc-gen`

Reply:
i don't understand this, lets talk irl